src/Repositories/CourseH5PProgressRepository.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace EscolaLms\Courses\Repositories;
+
+use EscolaLms\Courses\Models\H5PUserProgress;
+use EscolaLms\Courses\Models\Topic;
+use EscolaLms\Courses\Repositories\Contracts\CourseH5PProgressRepositoryContract;
+use Illuminate\Contracts\Auth\Authenticatable;
+
+class CourseH5PProgressRepository extends BaseRepository implements CourseH5PProgressRepositoryContract
+{
+    /**
+     * @var array
+     */
+    protected $fieldSearchable = [
+        'quiz_id',
```

Comment:
replace all `quiz` keyword with topic  with `topic` to follow names https://github.com/EscolaLMS/Courses/tree/main/doc

Reply:
changed